src/handler/get.ts

```diff
@@ -0,0 +1,42 @@
+import 'dotenv/config';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import logger from '../util/logger';
+import { addHttpHeaders } from '../util/httpHeaders';
+import { getGetErrors } from '../validators/get';
+import { getBySystemNumberAndCreatedTimestamp } from '../services/database';
+import { formatTechRecord } from '../util/formatTechRecord';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  logger.info('Get end point called');
+
+  const getErrors = getGetErrors(event);
+  if (getErrors) {
+    return addHttpHeaders(getErrors);
+  }
+
+  const systemNumber: string = decodeURIComponent(event.pathParameters?.systemNumber ?? '');
```

Comment:
Might want to validate the systemNumber and createdTimestamp are actually being sent (like the validate function we have on the search endpoint).

Reply:
NO